compiler-plugin/src/test/kotlin/arrow/meta/plugins/proofs/ResolutionTests.kt

```diff
@@ -51,8 +52,8 @@ class ResolutionTests {
       @Coercion
       internal fun String.toInt16(): Int? =
         toIntOrNull(16)
-        
-      val x: Int? = "30"
+  
+      val s = "30"
```

Comment:
I see this test is disabled.. but renaming `x` to `s` wont work since below is checking for the `x`